client/blocks/comments/post-comment-content.jsx

```diff
@@ -8,7 +8,11 @@ export default class PostCommentContent extends React.Component {
 	render() {
 		// Don't trust comment content unless it was provided by the API
 		if ( this.props.isPlaceholder ) {
-			return <div className="comments__comment-content">{ this.props.content }</div>;
+			return <div className="comments__comment-content">{
+				this.props.content.split( '\n' ).map( ( item, key ) => {
+					return <span key={ key }>{ item }<br /></span>;
```

Comment:
maybe return a `<p>{ item }</p>` for each item instead of a span with a br? That would be a bit more consistent with how comments end up getting formatted by wp.